### ISSUE TITLE
The admin app cannot process tickets with version 3.0.8

### ISSUE THREAD

**Issue by nicozanf** [web2py/web2py#2508]
web2py 3.0.8-stable+timestamp.2024.12.01.23.08.05
(Running on Rocket3 20241201.1, Python 3.12.7)

On Linux and Windows, the admin app is broken with version 3.0.8.

It correctly start and let you login, but trying to  modify an app hangs the servers without any error on console - and you have to kill the web2py server.

Also, clicking on a ticket is unuseful because it generates another ticket. Manually looking inside you could find this error:

```
\80\95\F5\AD\00\00\00\00\00\00}\94(\8Clayer\94\8C;/home/nico/web2py/applications/admin/controllers/default.py\94\8Ccode\94\8Ct<code object <module> at 0x788170282c10, file "/home/nico/web2py/applications/admin/controllers/default.py", line 1>\94\8Coutput\94\8C3<class 'NameError'> name 'to_native' is not defined\94\8C	traceback\94XM\00\00Traceback (most recent call last):
  File "/home/nico/web2py/gluon/restricted.py", line 214, in restricted
    exec(ccode, environment)
  File "/home/nico/web2py/applications/admin/controllers/default.py", line 2031, in <module>
  File "/home/nico/web2py/gluon/globals.py", line 474, in <lambda>
    self._caller = lambda f: f()
                             ^^^
  File "/home/nico/web2py/applications/admin/controllers/default.py", line 1837, in ticket
    traceback=(e.traceback and TRACEBACK(e.traceback)),
                               ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nico/web2py/applications/admin/controllers/default.py", line 1813, in __init__
    self.s = make_links(CODE(text).xml())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/nico/web2py/applications/admin/controllers/default.py", line 1791, in make_links
    link = make_link(lwords[i])
           ^^^^^^^^^^^^^^^^^^^^
  File "/home/nico/web2py/applications/admin/controllers/default.py", line 1774, in make_link
    return to_native(A('"' + tryFile + '"',
           ^^^^^^^^^
NameError: name 'to_native' is not defined
```

**Comment by nicozanf** [comment]
I've found that the 'modify' action is working. In fact is only the ERRORS action that's hanging. So, the problem seems strictly related to the tickets viewing.

**Comment by mbelletti** [comment]
It's possible that there are some old tickects? Maybe written with a different pickle protocol?

**Comment by nicozanf** [comment]
No, it happens on a brand new installation on a new folder

**Comment by nicozanf** [comment]
fixed with [#b1e3c78da6ad39fecf054ee6ba39aa50ef3d2a3b ](https://github.com/web2py/web2py/commit/b1e3c78da6ad39fecf054ee6ba39aa50ef3d2a3b)
